### PR TITLE
added splitwise api

### PR DESCRIPTION
**Pull request by shreyagupta30**

### CODE REVIEW

**Comment by satyammittal** [comment]
@shreyagupta30 Look into the pep8 issues listed above.

File: splitwise/config.py [hunk 1]
```diff
@@ -0,0 +1,2 @@
+consumer_key = '99WIaVrXvVaJDHYKAtZzZgFy145lSAE507PJFypd'
```
Comment (satyammittal) [inline splitwise/config.py]: You don't need to share keys on public platform. 
```suggestion
consumer_key = ''
```

**Comment by satyammittal** [comment]
Add a readme file in same folder explaining its usage.

**Comment by shreyagupta30** [comment]
@satyammittal okay, sir. On it. :+1:

File: splitwise/config.py [hunk 2]
```diff
@@ -0,0 +1,2 @@
...
+consumer_secret = 'koHlvnLMzVY4yMsNfj3IdBl6QWrSdxyuLTPuQLEK'
```
Comment (satyammittal) [inline splitwise/config.py]:
```suggestion
consumer_secret = ''
```

**Comment by satyammittal** [comment]
Also Steps to run it and where to get and setup the keys.

File: splitwise/config.py (hunk 1 above)
Comment (shreyagupta30) [reply splitwise/config.py]: Okay!

**Comment by satyammittal** [comment]
@shreyagupta30 Can we get integrated in our main chatbot code as well.

**Comment by shreyagupta30** [comment]
@satyammittal Sir, I searched for it a lot but didn't get much about it. The SDK available is just for a single POST request to create expense is available. Please give me 2 more days. I'll try doing it again and will get back to you.

**Comment by satyammittal** [comment]
@shreyagupta30 I am merging this. let me know if you get the new approach.

**Pull request merged**